cms/envs/test.py

```diff
@@ -320,7 +320,3 @@
 ######### custom courses #########
 INSTALLED_APPS += ('openedx.core.djangoapps.ccxcon',)
 FEATURES['CUSTOM_COURSES_EDX'] = True
-
-# API access management. Necessary so that django-simple-history
-# doesn't break when running pre-test migrations.
-INSTALLED_APPS += ('openedx.core.djangoapps.api_admin',)
```

Comment:
Actually, I forgot that tests can be skipped. I'm going to skip all the tests outside of LMS.